pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -223,6 +273,19 @@ rec {
     };
   };
 
+  bndtools = mirrorEclipsePlugins rec {
```

Comment:
This is added as an example of using the new system as intended, and also as a sanity check since it was the original motivating example which didn't work with the previous system.